dace/codegen/targets/cuda.py

```diff
@@ -424,8 +424,8 @@ def allocate_stream(self, sdfg, dfg, state_id, node, function_stream,
                     .format(**fmtargs), sdfg, state_id, node)
                 self._globalcode.write(
                     """
-DACE_EXPORTED void __dace_alloc_{location}(uint32_t size, dace::GPUStream<{type}, {is_pow2}>& result);
-void __dace_alloc_{location}(uint32_t size, dace::GPUStream<{type}, {is_pow2}>& result) {{
+DACE_EXPORTED void __dace_alloc_{location}(uint32_t {size}, dace::GPUStream<{type}, {is_pow2}>& result);
```

Comment:
Line 429 used `{size}` , however lines 427/428 used `size`. Thus the first argument to the `__dace_alloc_<...>()` function was called `size`, but line 428 used the actual name of the variable (for example `N`, or `nnz`), which was never actually declared, resulting in a compiler error.

Another option would be to use `size` everywhere in the function, but since a value for `{size}` was passed in by `.format()` I assumed this was the intended behaviour.

Reply:
Thanks!